CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ Changelog
 
 ### Fixed
 
-* Exception messages of comparisons between `DateTime(Immutalbe)` objects now display their date & time.
```

Comment:
I focused so much on whether or not this had one or two Ms that i missed the other typo 😅